Library/Homebrew/utils/gems.rb

```diff
@@ -241,6 +241,7 @@ def install_bundler_gems!(only_warn_on_failure: false, setup_path: true, groups:
 
     # Combine the passed groups with the ones stored in settings.
     groups |= (user_gem_groups & valid_gem_groups)
+    groups.delete("bootsnap") if HOMEBREW_PORTABLE_RUBY
```

Comment:
To clarify: I will remove it entire from the Gemfile in a separate PR. Looks like `homebrew-test-bot` unconditionally installs Bootsnap at the moment, which will need removed first.